.github/workflows/screener-build.yml

```diff
@@ -1,20 +1,29 @@
 name: Screener build
 
-on: workflow_dispatch
+on:
+  pull_request:
```

Comment:
At first we are simply migrating from ADO pipelines to GHA workflows, so the workflows will be triggered by PRs and pushes to master